core/tests/test_utils.py

```diff
@@ -481,18 +481,6 @@ def _get_unicode_test_string(self, suffix):
         """
         return '%s%s' % (self.UNICODE_TEST_STRING, suffix)
 
-    def setUp(self):
```

Comment:
I deleted this and the following functions as TestBase is only inherited once in the entire codebase and that is in AppEngineTestBase -- in AppEngineTestBase the functions (setup, teardown and _delete_all_models) are implemented